recipe/meta.yaml

```diff
@@ -29,8 +29,7 @@ requirements:
     - dask-expr >=1.1,<1.2
     - bokeh >=3.1.0
     - jinja2 >=2.10.3
-    - pyarrow >=7.0
-    - pyarrow-hotfix
+    - pyarrow >=14.0.1
```

Comment:
Seems like this change was actually made in `2024.8.2`, not sure if it's worth adding a patch in [conda-forge-repodata-patches-feedstock](https://github.com/conda-forge/conda-forge-repodata-patches-feedstock) to retroactively apply this new pinning to those older packages? cc @jakirkham

Reply:
Sounds like a good idea to me. Happy to review a repodata patch PR 🙂

Reply:
Thanks @jakirkham 😄 done in https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/886